Mlem/App/Views/Pages/Editors/FilterViolationWarning.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  FilterViolationWarning.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2025-02-03.
+//
+
+import SwiftUI
+
+struct FilterViolationWarning: View {
+    @Environment(Palette.self) var palette
+    
+    let failingText: String
+    let isPost: Bool
+    
+    var context: LocalizedStringResource {
+        isPost ? "Posts containing " : "Comments containing "
+    }
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: Constants.main.standardSpacing) {
+            (Text(Image(systemName: Icons.warning)) + Text(" Filter violation"))
```

Comment:
Ideally we'd avoid having a leading space in the localization string. Would a label work here?

```suggestion
            Label("Filter violation", systemImage: Icons.warning)
```